app/views/layouts/application.html.erb

```diff
@@ -9,7 +9,7 @@
   <body>
     <nav>
       <h1>Welcome to Hogwarts!</h1>
-      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", student_path %> | <%= link_to "Enroll", student_path %></h2>
+      <h2><%= link_to "View Houses", house_path %> | <%= link_to "View Students", students_path %> | <%= link_to "Enroll", students_path %></h2>
```

Comment:
`View Students` and `Enroll` are using the same path helperhere, but they should go to different pages. What is the purpose of the enroll page? What controller action should it map to? Use your answers to those questions to select the correct path helper from the output of `$ rails routes`